packages/docs/getting-started/running-a-validator.md

```diff
@@ -117,177 +133,170 @@ In all the commands we are going to see the `CELO_IMAGE` variable to refer to th
 docker pull $CELO_IMAGE
 ```
 
-### Create accounts
+### Create the Validator and Validator Group accounts
 
-At this point we need to create the accounts that will be used by the Validator, the Proxy and the attestation signer. We create and cd into the directory where you want to store the data and any other files needed to run your node. You can name this whatever you’d like, but here’s a default you can use:
+These are going to be the most important keys you'll generate. These are the keys that will control funds, participate in the PoS protocol and authorize other keys. They only need to be unlocked for when you submit transactions, so they can be unlocked on your local machine. Ideally they live on a hardware wallet. For this guide, we'll create the keys on a local node:
```

Comment:
First, you'll need to generate account keys for your Validator and Validator Group. These are the keys that will have access to your Celo Gold, and thus should be handled with care. For the purposes of this guide, we will be storing these keys on your local machine, but we recommend that you store then in a more secure manner.